src/app.tsx

```diff
@@ -112,7 +115,15 @@ function App(): JSX.Element {
   }, [fetchGlobalState]);
 
   useEffect(() => {
-    new LocalStorageProvider().saveApplicationVersion().then();
+    const checkVersions = async (): Promise<void> => {
+      const latestLocalVersion = CookiesProvider.getCookie("appversion");
+      const latestAppVersion = await getLatestAppVersion;
```

Comment:
Tutaj nie powinno być wywołanie funkcji?

Reply:
Wywołanie checkVersions() jest poniżej w tym useEffect

Reply:
Mi chodziło o `getLatestAppVersion`

Reply:
![obraz](https://user-images.githubusercontent.com/21079319/116405969-8482e700-a830-11eb-830f-6ad2dab4e73b.png)

Reply:
`getLatestAppVersion` to Promise zwracający wartość w returnie, wszystko tutaj działa tak jak powinno

Reply:
No chyba że powinienem to zrobić w jakiś inny sposób

Reply:
Dobra, racja nie patzyłem na implementację. 
Zmień proszę w takim razie nawzę na `latestAppVersion`, bo prefiks `get` nasuwa że to jest funkcja mimo tego że funkcją nie jest

Reply:
Zrobione